sass/directives/_tables.scss

```diff
@@ -178,3 +178,11 @@
     content: '\f00c'; // Checkmark icon
   }
 }
+
+@mixin section-bar {
+  background-color: $grey-lightest;
+  padding: $base-margin / 2;
+  margin-bottom: $section-padding;
```

Comment:
There might be more space than you intended. Maybe remove top padding on the first-child of the boxes

![image](https://cloud.githubusercontent.com/assets/2359538/22798472/cd59ddd4-eec7-11e6-98e5-506b5975d85c.png)